framework/console/controllers/FixtureController.php

```diff
@@ -415,13 +415,36 @@ private function findFixtures(array $fixtures = [])
         $foundFixtures = [];
 
         foreach ($files as $fixture) {
-            $foundFixtures[] = basename($fixture, 'Fixture.php');
+            $relativeName = $this->getFixtureRelativeName($fixture);
+            $foundFixtures[] = $relativeName;
         }
 
         return $foundFixtures;
     }
 
     /**
+     * Calculates $fixture's name relatively to $templatePath.
+     * Basically, strips getFixturePath() and 'Fixture.php' prefix from fixture's full path
+     * @see getFixturePath()
+     * @param $fixture string Full fixture path
+     * @return string Relative fixture name
+     *
+    */
+    private function getFixtureRelativeName($fixture) {
+        // $fixturesPath is normalized to unix format in getFixturesPath()
```

Comment:
The function argument should be more clear, in this case $fullFixturePath if the @param comment is correctly formulated.